src/lib/fcitx-utils/log.cpp

```diff
@@ -164,7 +153,26 @@ bool LogCategory::fatalWrapper2(LogLevel level) {
 const LogCategory &Log::defaultCategory() { return fcitx::defaultCategory(); }
 
 void Log::setLogRule(const std::string &ruleString) {
-    LogRegistry::instance().setLogRule(ruleString);
+    log_rules_t parsedRules;
+    auto rules = stringutils::split(ruleString, ",");
+    for (const auto &rule : rules) {
+        auto ruleItem = stringutils::split(rule, "=");
+        if (ruleItem.size() != 2) {
+            if (ruleItem.size() == 1 && ruleItem[0] == "notimedate")
```

Comment:
Suggest check special flag before split with "=". So if we want to have some more special values, it can be put together.

if (rule == "'...") {
     ....
} else if (rule == "...") {
    ....
} else {
}